drivers/mspi/mspi_esp32.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2025 Pavel Maloletkov.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/mspi.h>
+#include <zephyr/drivers/clock_control.h>
+
+#include <hal/spi_hal.h>
+
+#ifdef SOC_GDMA_SUPPORTED
+#include <hal/gdma_hal.h>
+#include <hal/gdma_ll.h>
+#endif
+
+#define MSPI_MAX_FREQ             80000000
+#define MSPI_TIMEOUT_MS           100
+#define SPI_DUTY_CYCLE_50_PERCENT 128
+
+/* Driver configuration structure */
```

Comment:
Nit, The config and data structures are self explanatory, so this comment can be safely removed.

Reply:
Done